universal-application-tool-0.0.1/app/views/admin/programs/ProgramIndexView.java

```diff
@@ -43,18 +40,21 @@ public Content render(ImmutableList<ProgramDefinition> programs, Http.Request re
                     .withClasses(Styles.INLINE_BLOCK)
                     .with(renderNewProgramButton(), maybeRenderPublishButton(programs, request)),
                 each(
-                    programs.stream()
-                        .collect(new GroupByKeyCollector<>(ProgramDefinition::adminName)),
-                    e -> this.renderProgramListItem(e.getKey(), e.getValue(), request)));
+                    programs.getProgramNames(),
+                    name ->
+                        this.renderProgramListItem(
+                            programs.getActiveProgramDefinition(name),
+                            programs.getDraftProgramDefinition(name),
+                            request)),
+                renderNewProgramButton(),
```

Comment:
Thought (blocking): this may also be from a merge conflict. These 2 lines have been moved above (line 41).

Reply:
Done!